.github/workflows/reusable-audit.yml

```diff
@@ -23,6 +23,11 @@ jobs:
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
         with:
           ref: ${{ matrix.ref }}
+      - name: Install asdf
+        uses: asdf-vm/actions/setup@707e84f3ee349548310aeabdad0dd3bfcb9b69fa # v1.1.0
+      - name: Add plugins
+        run: |
+         asdf plugin add yamllint https://github.com/ericcornelissen/asdf-yamllint
```

Comment:
This change does not properly take affect until after this Pull Request is merged as this re-usable workflow is always referenced from `main`. Hence, this check ("Check / Audit / Docker") will be failing.

Reply:
This was verified through the [continuous integration run](https://github.com/ericcornelissen/js-regex-security-scanner/actions/runs/3933661890/jobs/6727611839) for 2e6b66f19825d545448c60b535f56feb1fa3f77e.